README.md

```diff
@@ -33,6 +33,28 @@ Example of running cargo build on the kvm-ioctls crate:
     Finished release [optimized] target(s) in 5.63s
 ```
 
+## Testing Changes locally with the Container Image
+
+First build the rust-vmm container locally by running the commands
+
+```bash
+> cd rust-vmm-dev-container
+> ./docker.sh build
+```
+
+since this command will build a new docker image with tag latest version + 1
+and will alias it with "latest" tag, when testing the container use the latest
+tag that will be published with your PR to be sure that the changes introduced
+by your PR to the CI works correctly before pusing it upstream.
```

Comment:
You can use `./docker.sh print-next-version` to have that version. Maybe better to add it in the readme as an example.

Reply:
ok thanks!